tests/describe_task_queue.go

```diff
@@ -88,7 +88,7 @@ func (s *DescribeTaskQueueSuite) publishConsumeWorkflowTasksValidateBacklogInfo(
 	expectedBacklogCount[enumspb.TASK_QUEUE_TYPE_ACTIVITY] = 0
 	nullBacklogHeadCreateTime := true
 
-	tl := "backlog-counter-task-queue"
+	tl := s.randomizeStr("backlog-counter-task-queue")
```

Comment:
Think it's better to have tl names randomized since I was encountering an error where the tests were not passing cause of invalid outputs. More digging let to me realizing that this was the cause and is consistent with the way task queues are being created in other functional tests.